packages/cli/commands/fetch.js

```diff
@@ -82,5 +82,13 @@ exports.builder = yargs => {
     type: 'string',
   });
 
+  yargs.options({
```

Comment:
Is this something we want visible right away, or should it be hidden?

Reply:
Good catch, going to hide this.